static/tabs/home.html

```diff
@@ -15,8 +15,37 @@
                     muted
                     id="player"
                     poster="{{ url_for('static', filename='img/load-screen.svg') }}"
+                ></video>
```

Comment:
I'd really like to get rid of the last few unrelated changes.

It's not that I'm trying to be difficult, it's because they make merging and rebasing much more difficult.

So for example, this kind of change:

```
></video>
```
vs
```
>
</video>
```

Is particularly problematic. It's almost guaranteed to cause a merge conflict.

Compared to:

```
  <span id="progress" class="position-absolute w-100 fw-bold">
    0%
  </span>
```

vs

```
  <span id="progress" class="position-absolute w-100 fw-bold"> 0% </span>
```

These are a whole number of lines being replaced, so they are less "sharp", so to speak.

Anyway, I'm not doing it to annoy you, I'm doing it to not annoy me :laughing:

Reply:
✅